--- a/net/minecraft/client/resources/model/ModelBakery.java
+++ b/net/minecraft/client/resources/model/ModelBakery.java
@@ -190,7 +_,7 @@
                 Property<?> property = container.getProperty(string1);
                 if (property != null && iterator.hasNext()) {
                     String string2 = iterator.next();
-                    Comparable<?> valueHelper = getValueHelper((Property<Comparable<?>>)property, string2);
+                    Comparable<?> valueHelper = getValueHelper((Property<?>)property, string2);
                     if (valueHelper == null) {
                         throw new RuntimeException(
                             "Unknown value: '" + string2 + "' for blockstate property: '" + string1 + "' " + property.getPossibleValues()
